samples/core/xgboost_training_cm/xgboost_training_cm.py

```diff
@@ -227,7 +230,7 @@ def xgb_train_pipeline(
         region=region,
         name=cluster_name
     )):
-        create_cluster_op = dataproc_create_cluster_op(
+        create_cluster_task = dataproc_create_cluster_op(
```

Comment:
Is there any particular reason why you convert the _op to _task? I checked the other samples, _op has been consistently used.

Reply:
Actually no, just want to make things consistent without confliction. I can make it _op. Thanks for bringing it up.